pom.xml

```
@@ -12,7 +12,7 @@
     <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <name>Pipeline: Build Step</name>
-    <url>https://wiki.jenkins.io/display/JENKINS/Pipeline+Build+Step+Plugin</url>
+    <url>https://github.com/jenkinsci/pipeline-build-step-plugin</url>
```

[The archetype](https://github.com/jenkinsci/archetypes/blob/1b62176a257bb392f968b578e52818e6489936dc/hello-world/src/main/resources/archetype-resources/pom.xml#L81) uses `project.artifactId` here. While an extremely minor improvement, it reduces duplication and makes it easier to copy and paste examples from one POM to another.

```suggestion
    <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
```